src/environment.rs

```diff
@@ -642,7 +646,7 @@ async fn remove_package_from_environment(
     // Remove the conda-meta file
     let conda_meta_path = target_prefix.join("conda-meta").join(format!(
         "{}-{}-{}.json",
-        package.repodata_record.package_record.name,
+        package.repodata_record.package_record.name.as_source(),
```

Comment:
This one should be normalized

```suggestion
        package.repodata_record.package_record.name.as_normalized(),
```